### PR TITLE
Switch resolve command back to multi-process

### PR DESCRIPTION
**Pull request by shrutiparabgoogle**
Move ExecCommandTask back to using exec.Command()

### CODE REVIEW

File: cmd/registry/controller/local-tasks.go [hunk 1]
```diff
@@ -38,59 +27,21 @@ type ExecCommandTask struct {
...
 		return err
 	}
-	// Set args if any
-	if len(args) > 1 {
-		cmd.SetArgs(args[1:])
-	}
-	if err := cmd.Execute(); err != nil {
-		return fmt.Errorf("error executing action %s: %s", task.Action, err)
-	}
+	log.Printf("Finished executing taskId %s with action: %s\n Output: %s", task.TaskID, task.Action, output)
```
Comment (seaneganx) [inline cmd/registry/controller/local-tasks.go:44]: Here we can use `cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr` to hook the process directly into the correct outputs. The logs for each action will be interleaved without any way to identify them separately right now though.

No action necessary if you prefer the current output until we have unique identifiers attached to command logs.

**Review by seaneganx — COMMENTED** [review]
The workaround in `cmd/registry/cmd/resolve/resolve_test.go` can also be rolled back to test `Command(ctx)` directly instead of nesting it under a fake root command.

**Review by seaneganx — APPROVED** [review]
Approving since I'm on vacation for the first few days of next week. Please address the changes I suggested to `resolve_test.go` before submitting.

File: cmd/registry/controller/local-tasks.go (hunk 1 above)
Comment (shrutiparabgoogle) [reply cmd/registry/controller/local-tasks.go:44]: Noted. I am planning to take care of the logging change in a separate PR.

**Comment by shrutiparabgoogle** [comment]
> The workaround in `cmd/registry/cmd/resolve/resolve_test.go` can also be rolled back to test `Command(ctx)` directly instead of nesting it under a fake root command.

Done.

**Pull request merged**